Library/BillingInfo.cs

```diff
@@ -430,6 +430,15 @@ internal override void WriteXml(XmlTextWriter xmlWriter)
                 xmlWriter.WriteStringIfValid("phone", PhoneNumber);
                 xmlWriter.WriteStringIfValid("vat_number", VatNumber);
                 xmlWriter.WriteStringIfValid("currency", Currency);
+                xmlWriter.WriteStringIfValid("account_number", AccountNumber);
+                xmlWriter.WriteStringIfValid("type", Type);
+                xmlWriter.WriteStringIfValid("sort_code", SortCode);
+                xmlWriter.WriteStringIfValid("bsb_code", BsbCode);
+                xmlWriter.WriteStringIfValid("iban", Iban);
+                xmlWriter.WriteStringIfValid("paypal_billing_agreement_id", PaypalBillingAgreementId);
+                xmlWriter.WriteStringIfValid("amazon_billing_agreement_id", AmazonBillingAgreementId);
+                xmlWriter.WriteStringIfValid("amazon_region", AmazonRegion);
+                xmlWriter.WriteStringIfValid("external_hpp_type", ExternalHppType.Value.ToString().EnumNameToTransportCase());
```

Comment:
This call should probably stay the same. by calling `.Value` you are assuming it's always gonna be there. it's unsafe to operate on the value without first checking if it's null or not.